src/main/topLevelPages/myDashboard/latestContributions/myLatestContributions/MyLatestContributions.tsx

```diff
@@ -66,14 +67,20 @@ const MyLatestContributions = () => {
       <PageContentBlockHeader title={t('pages.home.sections.myLatestContributions.title')} />
       <ScrollerWithGradient>
         <Box padding={1}>
-          {activities?.map(activity => (
-            <ActivityViewChooser
-              key={activity.id}
-              activity={activity as ActivityLogResultType}
-              journeyUrl={activity.journey?.profile.url ?? ''}
-              footerComponent={MyActivityViewFooter}
-            />
-          ))}
+          {activities && activities?.length > 0 ? (
+            activities.map(activity => (
+              <ActivityViewChooser
+                key={activity.id}
+                activity={activity as ActivityLogResultType}
+                journeyUrl={activity.journey?.profile.url ?? ''}
+                footerComponent={MyActivityViewFooter}
+              />
+            ))
+          ) : (
+            <Box padding={1}>
```

Comment:
is this box needed? I think `<CaptionSmall padding={1}>` should look the same and the markup is simpler